tfe/resource_tfe_team_project_access.go

```diff
@@ -0,0 +1,162 @@
+package tfe
+
+import (
+	"context"
+	"errors"
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"log"
+)
+
+func resourceTFETeamProjectAccess() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceTFETeamProjectAccessCreate,
+		ReadContext:   resourceTFETeamProjectAccessRead,
+		UpdateContext: resourceTFETeamProjectAccessUpdate,
+		DeleteContext: resourceTFETeamProjectAccessDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: map[string]*schema.Schema{
+			"access": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(tfe.TeamProjectAccessAdmin),
+						string(tfe.TeamProjectAccessRead),
+					},
+					false,
+				),
+			},
+
+			"team_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringMatch(
+					projectIDRegexp,
+					"must be a valid project ID (prj-<RANDOM STRING>)",
+				),
+			},
+		},
+	}
+}
+
+func resourceTFETeamProjectAccessCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	tfeClient := meta.(*tfe.Client)
```

Comment:
Panic :exclamation: 

```suggestion
	config := meta.(ConfiguredClient)
	// you can use the tfe client: config.Client.Workspaces.Blah()
```
This change was introduced in #762 

In effect, `organization` attributes are no longer required since you can configure the provider with a universal org.

Reply:
Oops oops oops, I didn't realize I had pulled this in with a rebase. At it would have burned me in the merge anyway, so thanks for catching this! I applied that pattern to remove all usages of `meta.(*tfe.Client)`.